### DISCUSSION TITLE
How plugin do I use when working with Angular version 7 ?

### DISCUSSION THREAD

**Discussion by pablovegaimexhs** [eslint/eslint#14123] [Q&A]
I everybody, I am working on a project and I want to use **eslint**. When I was searching for a linter I see **tslint** but I also see that is deprecated:
https://palantir.github.io/tslint/
![image](https://user-images.githubusercontent.com/67170659/108539441-0218fc80-72ae-11eb-8067-464a82d61ecd.png)

So I want to know which plugin should I use since I am working with typescript using Angular version 7.

**Comment by mdjermanovic** [comment]
Hi @pablovegaimexhs!

You'll probably find all the info in this repository:

https://github.com/angular-eslint/angular-eslint
